timemachine/fe/topology.py

```diff
@@ -251,10 +276,9 @@ def parameterize_nonbonded_pairlist(self, ff_q_params, ff_lj_params):
                     q * sf_q,
                     sig,
                     eps * sf_lj,
-                    0.0,  # w offset
+                    0.0,  # w offset for intramolecular term
```

Comment:
Point of confusion: Meaning of intra molecular w offset. (Does increasing this number make "all intramolecular pairs further away from each other," or something else?)

Reply:
Yes, it increases the effective distance between pairs.